src/main/java/iudx/aaa/server/policy/EmailInfo.java

```diff
@@ -24,13 +29,22 @@ public class EmailInfo {
   Map<String, ResourceObj> itemDetails;
   Map<String, List<UUID>> providerIdToAuthDelegateId;
   Map<String, JsonObject> userInfo;
-  public EmailInfo(){
+  Map<String,String> expiryDurationMap;
 
+  public EmailInfo(){
+    this.expiryDurationMap = new HashMap<>();
   }
   public UUID getConsumerId() {
     return consumerId;
   }
-
+  public Map<String, String> getExpiryDurationMap() {
+    return expiryDurationMap;
```

Comment:
- May be better to return a copy of the map (`return new HashMap<String, String>(expiryDurationMap)`) instead of the actual map itself (I'm not 100% sure of this, see [here](https://dzone.com/articles/java-getter-and-setter-basics-common-mistakes-and))
- Can add a comment somewhere indicating that the key for this map is the item cat ID for the particular request